components/Zurueckbutton.js

```diff
@@ -0,0 +1,23 @@
+import styled from "styled-components";
+
+export default function ZurueckButton() {
```

Comment:
If your colleagues cannot speak german, they will not understand the meaning of this name